addons/binding/org.openhab.binding.pioneeravr/src/main/java/org/openhab/binding/pioneeravr/protocol/event/AvrUpdateListener.java

```diff
@@ -12,10 +12,11 @@
 
 /**
  * This interface defines interface to receive status updates from pioneerav receiver.
- * 
- * @author Antoine Besnard
- * @author Rainer Ostendorf
- * @author based on the Onkyo binding by Pauli Anttila and others
+ *
+ * Based on the Onkyo binding by Pauli Anttila and others
+ *
+ * @author Antoine Besnard - Initial contribution
+ * @author Rainer Ostendorf - Initial contribution
```

Comment:
It was part of the Initial contribution PR [AvrUpdateListener@#169](https://github.com/openhab/openhab2-addons/pull/169/files#diff-86f7a2f408aa2f7f38e25e65a7081157R16).

I could also remove the description from the second author since it is not really clear what kind of contribution was made. :-)

See also https://github.com/openhab/static-code-analysis/issues/199 for some background info on what should be there.